src/SDKs/SqlManagement/Sql.Tests/Utilities/TestEnvironmentUtilities.cs

```diff
@@ -9,6 +9,75 @@ namespace Sql.Tests
 {
     public static class TestEnvironmentUtilities
     {
+        private const string _environmentVariableName = "TEST_CSM_ORGID_AUTHENTICATION";
+
+        private static readonly TestEnvironment _environment = 
+            new TestEnvironment(Environment.GetEnvironmentVariable(_environmentVariableName));
+
+        // We now load default locations from environment variable.
+        // Tests were recorded with this appended to DefaultLocationId=japaneast;DefaultLocation=Japan East;DefaultSecondaryLocationId=centralus;DefaultSecondaryLocation=Central US;DefaultStagePrimaryLocation=North Europe;DefaultStageSecondaryLocation=SouthEast Asia;DefaultEuapPrimaryLocation=East US 2 EUAP;DefaultEuapPrimaryLocationId=eastus2euap
+        public static string DefaultLocation
+        {
+            get
+            {
+                return GetValueFromEnvironment("DefaultLocation");
```

Comment:
@nathannfan I am sorry if I said it this way. I apologize for the trouble.
But moving forward please consider your tests if they can run across regions/locations?
and hard coding is not going to help it.
If you guys are still fine with this approach, let's merge it.
When profile feature comes online, these tests if failed will have to be fixed.